packages/fx-core/src/plugins/resource/appstudio/index.ts

```diff
@@ -157,6 +157,23 @@ export class AppStudioPlugin implements Plugin {
     return this.appStudioPluginImpl.createManifestForRemote(ctx, maybeSelectedPlugins, manifest);
   }
 
+  public async scaffold(ctx: PluginContext): Promise<Result<any, FxError>> {
+    TelemetryUtils.init(ctx);
+    TelemetryUtils.sendStartEvent(TelemetryEventName.scaffold);
+    try {
+      TelemetryUtils.sendSuccessEvent(TelemetryEventName.scaffold);
```

Comment:
success event should be sent after scaffold succeed.

Reply:
Thanks! The telemetry will now be sent after scaffold succeed.